altair/vegalite/v5/api.py

```diff
@@ -23,8 +23,8 @@
     using_vegafusion as _using_vegafusion,
     compile_with_vegafusion as _compile_with_vegafusion,
 )
-from ...utils.data import DataType, is_data_type as _is_data_type
-from ...utils.deprecation import AltairDeprecationWarning
+from altair.utils.data import DataType, is_data_type as _is_data_type
+from altair.utils.deprecation import AltairDeprecationWarning  # noqa: F401
```

Comment:
I've kept this import to avoid merge conflicts. 

Ideally, `AltairDeprecationWarning` should be imported at the top level only, for end-users to add to filters.

Reply:
Can we now remove this line? I can't follow why it needs to be kept here

Reply:
> Can we now remove this line? I can't follow why it needs to be kept here

IIRC, removing this import also removed it from `altair.__init__.py` as it was the only explicit import of it. 

Its an easy fix, but I wanted to avoid messing with import across multiple files at the time (I think)

Reply:
> IIRC, removing this import also removed it from altair.__init__.py as it was the only explicit import of it.

Yeah it triggers this failure, I'll resolve with an explicit one further up

```
    def test_completeness_of__all__():
        relevant_attributes = update_init_file.relevant_attributes(alt.__dict__)

        # If the assert statement fails below, there are probably either new objects
        # in the top-level Altair namespace or some were removed.
        # In that case, run `hatch run update-init-file` to update __all__
>       assert alt.__all__ == relevant_attributes
E       AssertionError: assert ['Aggregate',...tString', ...] == ['Aggregate',...tString', ...]
E
E         At index 6 diff: 'AltairDeprecationWarning' != 'Angle'
E         Left contains one more item: 'with_property_setters'
E         Use -v to get more diff

tests\test_toplevel.py:12: AssertionError
========================== 1 failed, 1257 passed, 1 skipped, 3 xfailed, 1 xpassed, 8 warnings in 45.23s ===========================
```